src/Apps/__stories__/Apps.story.tsx

```diff
@@ -14,11 +15,22 @@ storiesOf("Apps", module)
     )
   })
   .add("Artist Page", () => {
+    const filterState = new FilterState()
+    // TODO: This currently only changes the UI, the first fetch isn’t taking
+    //       this state into account.
+    filterState.state.for_sale = true
```

Comment:
Setting up some initial state.

See comment about the initial state not yet adjusting fetch. Although I now realise that the initial state should pretty mirror what the server has already loaded, so this probably isn’t a real problem.